cli/stack.go

```diff
@@ -57,6 +57,23 @@ func CreateStack(c *cli.Context) {
 	}
 }
 
+func ChangeImage(c *cli.Context) {
+	checkRequiredFlagsAndArguments(c)
+	cbClient := NewCloudbreakHTTPClientFromContext(c)
+
+	var req models_cloudbreak.StackImageChangeRequest
```

Comment:
Would be better to declare where it is used, and would be more gopher to declare and initialize in one step